extensions/Gemini/python/src/aiconfig_extension_gemini/Gemini.py

```diff
@@ -16,6 +17,21 @@
 if TYPE_CHECKING:
     from google.generativeai.types import AsyncGenerateContentResponse
 
+
+DOCSTRING = """
+Model Parser for Gemini text to text models. Doesn't support image generation yet. 
+Function calling is not available on the public release Gemini api. Function Calling is also not supported in this model parser.
+See this link for more information on api versions: https://ai.google.dev/docs/api_versions
+
+@ankush-lastmile 
+TODO: This model Parser does not support multimodal
+TODO: This model Parser does not support function calling (not available)
+TODO: This model Parser does not support serializing all different types of the Gemini API (ie protos)
+TODO: This model Parser does not support strongly structuring the input data containing the configmetadata for the Gemini API
```

Comment:
The `GenerationConfig` params must be specified explicitly underneath `settings` as it's own nested dict (see "gH#1232 test plan for details")